src/test/resources/json/bounded_context_one.json

```diff
@@ -10,11 +10,11 @@
       "dependencies": [
         {
           "refersTo": "io.candydoc.ddd.sample.one_valid_bounded_context.feature_A.DomainCommandOne",
-          "allowed": true
+          "authorization": "ALLOWED"
```

Comment:
Keeping the "deterred" dependencies or not, this is breaking the API contract (json & yml outputs). This is a breaking change that would leads to a Candy-Doc v3.0.0 (also, take a look at the open close principle).